pages/common/exiftool.md

```diff
@@ -5,20 +5,20 @@
 
 - Remove all EXIF metadata from the given files:
 
-`exiftool -All= {{file}}`
+`exiftool -All= {{file1 file2 ...}}`
 
-- Increase time photo taken by 1 hour in directory:
+- Move the shooting date of all photos in a directory 1 hour forward:
 
-`exiftool "-AllDates+=0:0:0 1:0:0" {{directory}}`
+`exiftool "-AllDates+=0:0:0 1:0:0" {{path/to/directory}}`
 
-- Decrease time photo taken by 1 day and 2 hours on JPEGs only:
+- Move the shooting date of all JPEG photos in the current directory 1 day and 2 hours backward:
 
 `exiftool "-AllDates-=0:0:1 2:0:0" -ext jpg`
 
-- Change only DateTimeOriginal by -1.5 hours & do not keep backups:
+- Only change the `DateTimeOriginal` field subtracting 1.5 hours, without keeping backups:
 
 `exiftool -DateTimeOriginal-=1.5 -overwrite_original`
 
-- Rename all JPEGs according to a DateTimeOriginal recursively:
+- Recursively rename all JPEG photos in a directory based on the `DateTimeOriginal` field:
 
-`exiftool '-filename<DateTimeOriginal' -d %Y-%m-%d_%H-%M-%S%%lc.%%e {{directory}} -r -ext jpg`
+`exiftool '-filename<DateTimeOriginal' -d %Y-%m-%d_%H-%M-%S%%lc.%%e {{path/to/directory}} -r -ext jpg`
```

Comment:
What is the angle bracket for ?

Reply:
It's just how the command works, like a "redirection". For example, from the manual page:

    exiftool '-filename<%f_${model;}.%e' dir
          Rename all files in "dir" by adding the camera model name to the file name.  The semicolon after the tag
          name inside the braces causes characters which are invalid in Windows file names to be deleted from the
          tag value (see the -p option documentation for an explanation).

(here it says "adding" because `%f` is the filename)

Reply:
Ah okay